frontend/public/kubevirt/components/modals/start-stop-vm-modal.jsx

```diff
@@ -13,12 +15,36 @@ class StartStopVmModal extends PromiseComponent {
 
   submit(event) {
     event.preventDefault();
-    const patch = [{
-      op: 'replace',
-      path: '/spec/running',
-      value: this.props.start
-    }];
-    const promise = k8sPatch(this.props.kind, this.props.resource, patch);
+    let promise;
+    const vm = this.props.resource;
+
+    // handle PXE boot
+    const annotations = _.get(vm, 'metadata.annotations', {});
+    if (this.props.start && annotations[ANNOTATION_FIRST_BOOT]) {
+      if (annotations[ANNOTATION_FIRST_BOOT] === 'true'){
+        annotations[ANNOTATION_FIRST_BOOT] = 'false';
+      } else {
+        // find bootable disk and change boot order
+        const disks = _.get(vm, 'spec.template.spec.domain.devices.disks', []);
+        const bootableDisk = disks.find(disk => disk.bootOrder === 2);
```

Comment:
Should not happen, but still possible: the `bootableDisk` can be na array of >1 elements